scripts/tablets/topology.py

```
@@ -0,0 +1,1048 @@
+
+    def close(self) -> None:
+        self.cluster.shutdown()
+
+
+def parse_size(s: str) -> int:
+    """
+    Parses a size in bytes, optionally with a binary unit suffix (e.g. "1GiB", "512MiB", "1024").
+    """
+    m = re.fullmatch(r'\s*(\d+(?:\.\d+)?)\s*([KMGTP]i?B?)?\s*', s, re.IGNORECASE)
+    if not m:
+        raise argparse.ArgumentTypeError(f"Invalid size: '{s}'")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)